docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+    command: >
+      sh -c "python manage.py migrate &&
+             python manage.py wait_for_db &&
+             python manage.py runserver 0.0.0.0:8000"
```

Comment:
Command wait_for_db should be before migrate, because our db must be ready for migrations